src/app/index.js

```diff
@@ -33,6 +37,12 @@ const App = ({ history }) => {
     dispatch(watchListRetrieved());
   }, []);
 
+  useEffect(() => {
+    Object.values(MODULE_ASSETS_NAME_ID_MAP).forEach((id) => {
+      selectSchema(id, network);
```

Comment:
Please rename to something more appropriate like `retrieveSchemas` and do `Object.values(...)...` inside that function.